js/main.js

```diff
@@ -16,6 +49,104 @@ function apiRequest(search) {
   xhr.responseType = 'json';
   xhr.addEventListener('load', getMovieData);
   function getMovieData() {
+    var ulElement = document.querySelector('ul');
+    for (var i = 0; i < xhr.response.Search.length; i++) {
+      var domTree = renderSearchResults(xhr.response.Search[i]);
+      ulElement.appendChild(domTree);
+      apiRequestID(xhr.response.Search[i].imdbID);
+    }
   }
```

Comment:
I don't recommend defining functions within other functions like this. I assume you are doing this because you want access to the response of the XHR object. You should make this function definition global and give it a parameter, then pass the response as an argument when you call the function.